cognite_toolkit/_cdf_tk/commands/modules.py

```diff
@@ -392,14 +396,22 @@ def upgrade(self, organization_dir: Path, verbose: bool = False) -> Changes:
 
     @staticmethod
     def _get_module_version(project_path: Path) -> Version:
-        if (system_yaml := project_path / "_system.yaml").exists():
+        cdf_toml = CDFToml.load()
+        # After `0.3.0` the version is in the CDF TOML file
+        if cdf_toml.is_loaded_from_file and cdf_toml.modules.version:
+            return parse_version(cdf_toml.modules.version)
+
+        elif (system_yaml := project_path / "_system.yaml").exists():
             # From 0.2.0a3 we have the _system.yaml on the root of the project
             content = read_yaml_file(system_yaml)
-        elif (system_yaml := project_path / BUILTIN_MODULES / "_system.yaml").exists():
+        elif (system_yaml := project_path / "cognite_modules" / "_system.yaml").exists():
```

Comment:
This must be hardcoded as it searches previous releases for the versions. And previous releases will not change.